token.json

```diff
@@ -0,0 +1 @@
+{"token": "ya29.a0AcM612wCcY-WiLKtB1L9SkV81synGisbF2c5YXkRGgJRqoERnYInNW_Gsyiy-kt7bRJKs3TSd-w8k43GG4nFwKCZABOpn4mYXDEN3sz4psQyNoLfpfMWlL3yxvadSHuIuxsxnmKwQR12IydiJsZ7iI1ROlWLauoKhLikI5c8tgaCgYKAecSARMSFQHGX2MirZuMf4uaErBWiI6Ik0rPkQ0177", "refresh_token": "1//0g1SjA2sfKBISCgYIARAAGBASNwF-L9Iru04lSAOq3M1iEatptE1qKZhpqGaZlDqbVHLNMAY9EsqMIPJpeBlk22lEzelcbgk2snE", "token_uri": "https://oauth2.googleapis.com/token", "client_id": "20477689782-q9gvbvj5si3v44g414qqlp4pa21bci95.apps.googleusercontent.com", "client_secret": "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", "scopes": ["https://www.googleapis.com/auth/gmail.send"], "universe_domain": "googleapis.com", "account": "", "expiry": "2024-10-15T11:34:32.567581Z"}
```

Comment:
its not good practise to push the secret striaght away to github, setup a env file